lib/support/chrome-perflog-parser.js

```diff
@@ -140,6 +141,15 @@ module.exports = {
             log.error('Error parsing response: ' + JSON.stringify(data, null, 2));
             throw e;
           }
+
+          let previousEntry = previousConnectionEntries.get(data.response.connectionId);
+          if(previousEntry) {
+            let startedDateTime = moment(entry.startedDateTime).valueOf();
+            startedDateTime += (previousConnectionEntries.get(data.response.connectionId));
+            entry.startedDateTime = moment.unix(startedDateTime / 1000).toISOString();
```

Comment:
Yep I need to circle back around on this as I was getting some errors and finally got it working with the above lines of code, definitely needs to be cleaned up and refactored. :-p I also discovered a bug with the first of the requests that reuse the initial connection id, so this may not be the final solution. I just wanted to get it in front of you so you knew the direction I was headed for a solution. ;-) I'm still waiting to hear back from the chrome IRC channel to see if anyone has any ideas on if this approach is valid or if there is an easier way to calculate blocking time.